packages/terra-site/src/examples/alert/ActionExample.jsx

```diff
@@ -35,7 +35,7 @@ class ActionExample extends React.Component {
           </code>
         </div>
         <br />
-        <Alert type={Alert.Opts.Types.WARNING} action={<Button text="Action" size="medium" variant="primary" onClick={this.actionFunc} />} >
+        <Alert type={Alert.Opts.Types.WARNING} action={<Button text="Action" size="medium" variant="emphasis" onClick={this.actionFunc} />} >
```

Comment:
If you are using Button v2, it won't have the size prop anymore right?

Reply:
Oops. No it won't.